build/aca-machine-init.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+BUILD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "build path is $BUILD"
+
```

Comment:
this is good, it should work well in our lab environment also.

Reply:
yes, we will have different environments for teams